lib/tasks/quality.rake

```diff
@@ -1,13 +1,65 @@
-require 'quality/quality_score'
+require 'quality/quality_score_calculator'
 
 namespace :quality do
 
   desc "Quality score for named dataset"
   task :score,[:dataset] => :environment do |_, args|
     dataset = Dataset.find_by(name: args.dataset)
-    q = QualityScore.new(dataset)
-    p q.score
-    p q.reasons
+    q = QualityScoreCalculator.new(dataset)
   end
 
+  desc "Calculate scores for an organisation"
+  task :calculate, [:organisation] => :environment do |_, args|
```

Comment:
Just so we don't have to bug you once you're gone, might be worth making a comment that it's expecting short-name or something.